src/canopy/utils/openai_exceptions.py

```diff
@@ -1,5 +1,12 @@
 import openai
 
+from importlib.metadata import version
```

Comment:
This can be deleted, we're fixing the openai dependency in #171.  
We don't intend to be backwards supporting.